### ISSUE TITLE
Support for `max_metadata_size` in grpc-js

### ISSUE THREAD

**Issue by 97amarnathk** [grpc/grpc-node#2050]
### Problem description
Is `grpc.max_metadata_size` supported in `grpc-js` ? 
If so, then what is the maximum allowed value for this ?

### Environment
 - OS name, version and architecture: [e.g. MacOS Monterey 12.0.1]
 - Node version 14.17.4
 - Node installation method nvm
 - Package name and version `@grpc/grpc-js:^1.5.5`

### Additional context
* I have migrated from "grpc" to "grpc-js".
* On going through the interop tests [here](https://github.com/grpc/grpc-node/pull/1571/files#diff-7aacea1567fae9a30c435061805ffe339e568a744b150f1ef53625037f8238e6R80),  I noticed that `grpc.max_metadata_size` is being used.
* However the documentation for `grpc-js`at npm does not list `grpc.max_metadata_size` as a valid channel option.
* This is also not mentioned in `ChannelOptions` type.

**Comment by murgatroid99** [comment]
No, that option is not supported. No limit is enforced in grpc-js, so the practical limit is whatever maximum value Node itself supports. The test in the linked PR applies to both libraries so it needs to set the option for the other library. However, that test is also currently skipped because it does not work.

**Comment by 97amarnathk** [comment]
Okay. 
Also a related doubt. Is GRPC metadata passed as a http header in the first frame ?

**Comment by murgatroid99** [comment]
gRPC metadata is transformed directly into HTTP/2 headers. There are some headers that gRPC itself sets, so you can't go right to the limit with metadata, but you can get close. The specific framing happens at a lower level and probably doesn't meaningfully affect the limit.

**Comment by darkLord19** [comment]
@murgatroid99 so how should one transport large metadata which exceeds this limit?
